empyrical/stats.py

```diff
@@ -727,6 +727,39 @@ def tail_ratio(returns):
         np.abs(np.percentile(returns, 5))
 
 
+def SQN(trade_returns):
+    """Determines the SystemQualityNumber of trades as defined by Van K. Tharp.
+
+    Parameters
+    ----------
+    trade_returns : pd.Series
+        A series of transaction gains/losses.
+
+    Returns
+    -------
+    float
+        SystemQualityNumber
+
+    """
+    n_trades = len(trade_returns.values)
+
+    if n_trades < 30:
+        return np.nan
+
+    pos = trade_returns[trade_returns > 0]
+
+    if len(pos) < 3:
+        return np.nan
+
+    std = np.nanstd(pos, ddof=1)
+    avg = np.nanmean(pos)
+
+    if abs(std) < np.finfo(np.float).eps:
```

Comment:
At first I ran into some errors with precision, so that was added. Added it for a sanity check. Changing it back to std==0..